extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php

```diff
@@ -0,0 +1,111 @@
+<?php
```

Comment:
the name of the file has an uppercase i, it should be `verifyWithEmptyDB.php ` not `verifyWIthEmptyDB.php `